src/GRA.Domain.Repository/INewsCategoryRepository.cs

```diff
@@ -10,5 +11,6 @@ public interface INewsCategoryRepository : IRepository<NewsCategory>
         Task<NewsCategory> GetDefaultCategoryAsync(int siteId);
         Task<ICollection<NewsCategory>> GetAllAsync(int siteId);
         Task<DataWithCount<IEnumerable<NewsCategory>>> PageAsync(BaseFilter filter);
+        Task SetLastPostDate(int categoryId, DateTime? date);
```

Comment:
It looks like you're calling this with `_dateTimeProvider.Now` which means you can use a type of `DateTime` rather than `DateTime?` which is a nullable `DateTime`.